### DISCUSSION TITLE
Unblock Dalgerok

### DISCUSSION THREAD

**Discussion by hackathonius** [community/community#132277] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi, my account Dalgerok was banned and there is 2 months of inactivity in my reinstation ticket. Can anybody look into it? Thanks in advance.

**Comment by ebndev** [comment]
Hi @aorap-hackathon,

This community is managed by GitHub employees, but it is not a support forum. You have done everything necessary on your end by opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
